src/script/conversation/ConversationVerificationStateHandler/MLS/MLSStateHandler.ts

```diff
@@ -58,12 +58,12 @@ export class MLSConversationVerificationStateHandler {
   ) {
     this.logger = getLogger('MLSConversationVerificationStateHandler');
     // We need to check if the core service is available and if the e2eIdentity is available
-    if (!this.core.service?.mls || !this.core.service?.e2eIdentity) {
+    if (!this.core.hasMLSDevice || !this.core.service?.e2eIdentity) {
```

Comment:
is this correct? If so, it doesn't match the code comment anymore right?

Reply:
indeed the comment needs updating, good catch :) 
The fix is needed though (we should never check if mls is enabled by checking that the service is there, anymore)

Reply:
gotcha, I guess we're just not spying on that `hasMLSDevice` in the unit test then